models/public/mozilla-deepspeech-0.8.2/pbmm_to_pb.py

```diff
@@ -0,0 +1,105 @@
+#! /usr/bin/env python3
+#
+# Copyright (C) 2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+import argparse
+
+try:
+    from tensorflow import GraphDef
+except ImportError:
+    from tensorflow.compat.v1 import GraphDef
+
+from mds_convert_utils.memmapped_file_system_pb2 import MemmappedFileSystemDirectory
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="Convert memory-mapped Tensorflow protobuf format into plain Tensorflow protobuf (GraphDef)")
+    parser.add_argument('input', type=str, help="Input .pbmm filename")
+    parser.add_argument('output', type=str, help="Output .pb filename")
+    parser.add_argument('--overwrite', action='store_true', help="Overwrite output file if exists")
+    return parser.parse_args()
+
+
+def bytes_to_uint(data):
+    res = 0
+    for digit in reversed(data):
+        res = (res << 8) + digit
+    return res
+
+
+def load_memmapped_fs(data):
+    if isinstance(data, str):
```

Comment:
Why is there an `if` here? Shouldn't you already know what type `data` is?

Reply:
Moved loading outside this function, now `data` is `bytes` and there's no `if`.